src/DIRAC/WorkloadManagementSystem/Utilities/CloudBootstrap/checkdisk.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import sys
+import os
+from subprocess import Popen, PIPE
+
+
+def getstatusoutput(cmd):
+  try:
+    inst = Popen(cmd, stdout=PIPE, stderr=PIPE)
+    output, _ = inst.communicate()
+    status = inst.returncode
+    return status, output
+  except EnvironmentError:
+    # Missing executable
+    return -1, ""
+
+
+def getMountedPartitions():
+
+  status, output = getstatusoutput(['disk', '-h'])
+
+  partitionList = []
+  for line in output.split('\n'):
+    if line.startswith('/dev'):
+      part = os.path.basename(line.split()[0])
+      size = line.split()[1].rstrip('G')
+      mount = line.split()[5]
+      partitionList.append((part, size, mount))
+
+  return partitionList
```

Comment:
I think that this is not py3-safe, but anyway you can probably get the same result with `psutil.disk_partitions()`

Reply:
And other methods of `psutil` can probably be useful here.

Reply:
I don't think this script is used. I couldn't find any references to it, so I've removed it.